Vagrantfile

```diff
@@ -0,0 +1,26 @@
+Vagrant.configure(2) do |config|
+    config.vm.box = "fedora/25-cloud-base"
+    config.vm.network "forwarded_port", guest: 9090, host: 9090
```

Comment:
Good point. Let's stick with the default, it's not that difficult to change the Vagrantfile if necessary.